apps/src/storage/dataBrowser/DataTableView.jsx

```diff
@@ -128,10 +128,11 @@ class DataTableView extends React.Component {
   };
 
   exportCsv = () => {
+    const isCurrent =
```

Comment:
Nit: I'm assuming this references what we think of as 'current tables'. However, the variable name can be confusing and can be interpreted as assessing whether a table is out-of-date or not.

Reply:
good point, thanks. Renamed to `isSharedTable`